src/Azure.DataApiBuilder.Mcp/BuiltInTools/CreateEntityTool.cs

```diff
@@ -8,15 +8,15 @@
 
 namespace Azure.DataApiBuilder.Mcp.BuiltInTools
 {
-    public class CreateRecordTool : IMcpTool
+    public class CreateEntityTool : IMcpTool
```

Comment:
@souvikghosh04 - it is `create-record` and `execute-entity`. NOT `create-entity`